fjord/feedback/views.py

```diff
@@ -45,8 +46,15 @@ def download_firefox(request, template):
 
 
 def thanks(request):
-    template = 'feedback/thanks.html'
-    return render(request, template)
+    user_opinion = Response.objects.get(id=request.session['opinion_id'])
+    length = len(user_opinion.description.split())
+
+    if waffle.flag_is_active(request, 'thankyou') and user_opinion.locale == 'en-US' and user_opinion.happy == False and length >= 7:
+        template = 'feedback/thanks.html'
```

Comment:
I was thinking we'd copy the `feedback/thanks.html` template to `feedback/thanks_sad.html`. That way we can write a series of tests to make sure this works correctly using `assertTemplateUsed`. Otherwise we have no way of testing that this code is working correctly.